apps/etterlatte-behandling/src/main/kotlin/behandling/klage/KlageService.kt

```diff
@@ -520,6 +520,10 @@ class KlageServiceImpl(
 
         val oppdatertKlage = klage.attesterVedtak()
 
+        val utfall = klage.utfall as KlageUtfallMedData.Avvist
```

Comment:
```suggestion
        val utfall = checkNotNull(klage.utfall as? KlageUtfallMedData.Avvist) {
          "Vi har en klage som kunne attesteres, men har feil utfall lagret. Id: $klageId"
        }
```
evt, for å ha en mer passende feilmelding hvis noe ille skjer (selv om det ikke skal skje da)